templates/_includes/text-row.html

```diff
@@ -0,0 +1,9 @@
+<div class="component-row">
```

Comment:
What is this file for? It's not referenced anywhere here in the PR. Please either use it or remove it.